src/content-cli-push.ts

```diff
@@ -165,8 +164,14 @@ class Push {
             .command("data-pools")
             .description("Command to push data pools")
             .option("-p, --profile <profile>", "Profile which you want to use to push the data pools")
+            .option("-f, --file <file>", "The file with the json request you want to push")
+            .option(
+                "-bi, --batchImport <batchImport>",
```

Comment:
We can just use --batchImport without the <batchImport>. This will be true if the user provides the flag false otherwise.

Reply:
Also update doc to reflect this change.

Reply:
thank you. Good point. I adjusted.